olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -336,7 +338,7 @@ void NetworkCurl::Deinitialize() {
     event_condition_.notify_all();
 #if defined(OLP_SDK_NETWORK_HAS_PIPE) || defined(OLP_SDK_NETWORK_HAS_PIPE2)
     char tmp = 1;
-    if (write(pipe_[1], &tmp, 1) < 0) {
+    if (write(pipe_[1u], &tmp, 1u) < 0u) {
```

Comment:
Nothing can be lower then 0 unsigned. The return of write() is signed `ssize_t` and is in case of an error negative. So you have now a signed vs. unsiged comparisson which is bad.